Validate number of fishers

efcaguab opened the pull request:
The purpose of this pull request is to add the number of fishers into the pipeline so it can be used in the portal. 

**New features**

- New alert for outlier in number of fishers in airtable (alert number 18)
- New columns in the public trips data frame, namely `fisher_n_man`, `fisher_n_woman`, and `fisher_n_child`
- New columns in the public aggregated data frame, namely  `prop_landings_woman` with the proportion of landings where a woman was recorded

**Issues fixed**

- Added other columns that have been validated but were not included in the clean trips_table data frame that's exported

langbart reviewed (changes requested):
Hey @efcaguab, I have just updated with main and added site and fishers alerts in flag  handling

efcaguab commented:
> Hey @efcaguab, I have just updated with main and added site and fishers alerts in flag handling

Thanks for tidying it up!!! Appreciate it! I'll go ahead and merge it.
